torch/nn/modules/conv.py

```diff
@@ -53,6 +53,20 @@ def reset_parameters(self):
             bound = 1 / math.sqrt(fan_in)
             init.uniform_(self.bias, -bound, bound)
 
+    def _compute_padding_same(self, input, dim):
```

Comment:
This means that users of `torch.nn.functional.conv2d` won't be able to use the `"same"` padding?